### DISCUSSION TITLE
Welcome to TAFE22S204 Discussions!

### DISCUSSION THREAD

**Discussion by SimonWJohnson** [SimonWJohnson/TAFE22S204#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SimonWJohnson** [comment]
Hi Team!

I hope you all had a great weekend :)

I've got some XAML code written up and ready to go for tomorrow.

See you then!

Simon

**Reply by CongHauLam** [reply]
Thank Simon, see you tomorrow.

**Comment by 000976998** [comment]
Hi Team,
About the UWP design, Do we need to follow the default design or I can change?
Thanks.

**Reply by SimonWJohnson** [reply]
Hey Mostafa,

with the XAML code, feel free to layout a design that you think looks good

:)

**Reply by CongHauLam** [reply]
Hi Mostafa,
Awesome, all your XAML and C# look so good.
Thanks;

**Comment by 000976998** [comment]
Hi Team,
Using the percentage symbol in "Interest rate" round up the digits to only 3 (0.00029... == 0.00%).
Do we need to have the Percentage Symbol?

**Reply by SimonWJohnson** [reply]
I'm not too sure, but if we get stuck (ibn terms of formatting), we could incorporate the % symbol in the xaml code to avoid any conflicts/crashes etc in the C# code

**Comment by SimonWJohnson** [comment]
Hey Mostafa,

How are you travelling with the Mortgage Calculator C# code?

**Reply by 000976998** [reply]
The design is done. There are still some work required for the data type.

**Comment by SimonWJohnson** [comment]
Hey Cong, Hey Mostafa, 

I've got some working C# code for the Currency Converter! Could you guys have a look and let me know what you think?

**Reply by CongHauLam** [reply]
Hi Simon,
XAML and C# code are super cool. Thank you for your effort.

**Reply by SimonWJohnson** [reply]
Thanks Cong!

**Comment by CongHauLam** [comment]
hi Team, 
I finished the XAML and C# code of Shopping_Cart. And I also sent Shopping_Cart.Zip to all guys. Please have a look with that and give me your feedback to improve it.

**Reply by SimonWJohnson** [reply]
Hey Cong, Looks really good, well done!  Very colourful :)

**Reply by 000976998** [reply]
looks good. Thanks.

**Comment by 000976998** [comment]
Hi Team,
A XAML scale responsive page can be time consuming. Can we use "Float Left" method for the design?

**Reply by SimonWJohnson** [reply]
We have a fair bit of freedom to move I believe, so as far as I'm concerned, if it works it works :)

**Comment by 000976998** [comment]
Hi Team,
The Mortgage Calculator Branch now has MERGED to the Development Branch.

**Reply by SimonWJohnson** [reply]
Nice one Mostafa!

**Comment by SimonWJohnson** [comment]
Hi Team,

I'm about to merge the Currency Converter branch into Development
